tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -184,8 +185,8 @@ void RunInference(Settings* s) {
       exit(-1);
   }
 
-  profiling::Profiler* profiler = new profiling::Profiler();
-  interpreter->SetProfiler(profiler);
+  auto profiler = absl::make_unique<profiling::Profiler>(s->max_profiling_buffer_entries);
```

Comment:
CC @renjie-liu 

This seems to be failing internal tests. `max_profiling_buffer_entries` was added to `BufferedProfiler`, not the `NoopProfiler`, which gets instantiated when the `TFLITE_PROFLING_ENABLED` flag is not true [here](lite).

@freedomtan , could you add a constructor to [`NoopProfiler`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/profiling/noop_profiler.h#L29) that accepts `max_profiling_buffer_entries`?